specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2021-11-01-preview/examples/GetMarketplaceRegistrationDefinitions.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "scope": "subscription/0afefe50-734e-4610-8a82-a144ahf49dea",
```

Comment:
hope not care about the validity of guid here - h in guid.

Reply:
These are randomly generated, the GUID validity doesn't matter for swagger purposes